dspace-api/src/main/java/org/dspace/core/AbstractHibernateDAO.java

```diff
@@ -10,21 +10,26 @@
 import java.sql.SQLException;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Map;
 import java.util.UUID;
+import javax.persistence.Query;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Expression;
+import javax.persistence.criteria.Root;
 
 import org.apache.commons.collections.CollectionUtils;
-import org.hibernate.Criteria;
-import org.hibernate.Query;
 import org.hibernate.Session;
-import org.hibernate.criterion.Projections;
+
+//import org.hibernate.query.Query;
```

Comment:
Do we need this?

Reply:
There is indeed no reason for that import to be sat there in comments. I will remove this line entirely in the next commit.